### DISCUSSION TITLE
version question

### DISCUSSION THREAD

**Discussion by alsnat** [monocongo/climate_indices#568] [Q&A]
I noticed that your project mentions compatibility with Python 3.8/3.9/3.10/3.11. I’m currently using Python 3.13, and I’m wondering if your project supports this latest version. Could you please let me know if there are any plans to update the compatibility or if there are specific instructions for using Python 3.13?

Thank you for your help!

**Comment by monocongo** [comment]
Yes, we can go to 3.13, please see the PR #569 . This will break things for users who need 3.8 or 3.9 so I'm not convinced we want this yet.
